sdk/scala-sdk-testkit/src/main/scala/com/akkaserverless/scalasdk/testkit/impl/TestKitEventSourcedEntityContext.scala

```diff
@@ -18,12 +18,20 @@ package com.akkaserverless.scalasdk.testkit.impl
 
 import akka.stream.Materializer
 import com.akkaserverless.scalasdk.eventsourcedentity.EventSourcedEntityContext
+import com.akkaserverless.scalasdk.eventsourcedentity.CommandContext
+import com.akkaserverless.scalasdk.Metadata
 
 /**
  * INTERNAL API Used by the generated testkit
  */
-final class TestKitEventSourcedEntityContext(override val entityId: String)
+final class TestKitEventSourcedEntityContext(
```

Comment:
repurposed this class to be able to be used when setting the CommandEvent with `entity._internalSetCommandContext`